src/components/common/Hooks/DrawerHooks.js

```diff
@@ -0,0 +1,34 @@
+import { createContext, useState } from 'react';
+
+export const ContextApi = createContext();
+
+export function DrawerProvider({ children }) {
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [createdTime, setCreatedTime] = useState('');
+  const [treeId, setTreeId] = useState('');
+  const [verifiedToken, setVerifiedToken] = useState('');
+  const [verifiedTree, setVerifiedTree] = useState('');
+
+  return (
+    <ContextApi.Provider
+      // eslint-disable-next-line react/jsx-no-constructed-context-values
+      value={{
+        firstName,
+        setFirstName,
+        lastName,
+        setLastName,
+        createdTime,
+        setCreatedTime,
+        treeId,
+        setTreeId,
+        verifiedToken,
+        setVerifiedToken,
+        verifiedTree,
+        setVerifiedTree,
+      }}
+    >
+      {children}
+    </ContextApi.Provider>
```

Comment:
It feels a bit like invasive code for me, Stanly, why you need to use this context.

Reply:
to make the data available  for the Drawer title component .